src/services/types/Config.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
The file is called Config, but it stores NotionConfig. Is it designed to be an abstract service config interface or notion-specific?

I'd suggest improving a directory structure a bit:

```
src/
  service/
    notion/
      index.ts - Notion Service
      config.ts - Notion Service Config interface
    service.ts - abstract service
  types/
    app-config.ts
  form.ts
  app.ts
```